### ISSUE TITLE
syslog RFC 5424 (TCP+TLS) appender RemoteSyslog5424Appender

### ISSUE THREAD

**Issue by mrdavidlaing** [cityindex/log4net.Appenders.Contrib#1]
log4net ships with a [log4net.Appender.RemoteSyslogAppender](https://logging.apache.org/log4net/release/sdk/log4net.Appender.RemoteSyslogAppender.html); which only supports the older syslog RFC 3164 over UDP protocol.  

To ship data to remote Logsearch ingestors over the public internet we need a similar appender that:

1.  Supports the newer [RFC 5424 syslog protocol](http://tools.ietf.org/html/rfc5424)
2.  Uses TCP
3.  Is secured with TLS

**Comment by mrdavidlaing** [comment]
https://github.com/merchantwarehouse/syslog4net looks like a  good place to start - it seems to support syslog 5424 over TCP; and just be missing the TLS encryption.

**Comment by mrdavidlaing** [comment]
Beta version implemented by 1278bdc0d06a70597ba8079456fed30ef7d7c256

Use:
```
<appender name="RemoteAppender" type="log4net.Appenders.Contrib.RemoteSyslog5424Appender,log4net.Appenders.Contrib">
      <AppName>log4net.Appenders.Contrib.SampleApp</AppName>
      <Server></Server>
      <Port></Port>
      <CertificatePath></CertificatePath>
      <layout type="log4net.Layout.PatternLayout">
        <conversionPattern value="%message" />
      </layout>
    </appender>
```
